src/client/AM.PA.MacOs/PersonalAnalytics/Trackers/EmotionTracker/EmotionTracker.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  EmotionTracker.swift
+//  PersonalAnalytics
+//
+//  Created by Luigi Quaranta on 2019-01-02.
+//
+
+import Cocoa
+import Foundation
+import CoreData
+
+struct Questionnaire {
+    var timestamp: NSDate
+    var activity: String
+    var valence: NSNumber
+    var arousal: NSNumber
+}
+
+class EmotionTracker: ITracker {
+
+    // MARK: Properties
+    let emotionPopUpController = EmotionPopUpWindowController(windowNibName: NSNib.Name(rawValue: "EmotionPopUp"))
+    let notificationCenter = NSUserNotificationCenter.default
+    var notificationSet = Set<NSUserNotification>()
+
+    // Properties for protocol conformity
+    var name: String = "EmotionTracker"
+    var isRunning: Bool = false
+
+
+    // MARK: Initializer
+    init() {
+        
+        // Set default time interval between notificaitons
+        var minutes = 60
+        minutes *= 60
+        UserDefaults.standard.set(minutes, forKey: "timeInterval")
+
+        // Start the tracker
+        start()
+    }
+
+
+    // MARK: ITracker Protocol Conformity Functions
+    func start() {
+        // Schedule first notification
+        scheduleNotification()
+        isRunning = true
+    }
+
+    func stop() {
+        // Cancel scheduled notification
+        for notification in notificationSet {
+            notificationCenter.removeScheduledNotification(notification)
+        }
+        isRunning = false
+    }
+
+    // Not yet implemented
+    func createDatabaseTablesIfNotExist() {}
```

Comment:
I would appreciate if you could implement this function, it doesn't really do anything yet but will make things a bit easier for me in the future 🙂 .